build.sbt

```diff
@@ -1,9 +1,15 @@
 name := "circe-yaml"
-version in ThisBuild := "0.3.1"
+version in ThisBuild := "0.4.0-SNAPSHOT"
```

Comment:
I'm not a fan of proliferating several build files, but if it is a useful convention I'm OK with a versions.sbt